action/rename.php

```diff
@@ -112,6 +113,53 @@ public function handle_ajax(Doku_Event $event) {
         }
     }
 
+    /**
+     * Handle media renames in media manager
+     *
+     * @param Doku_Event $event
+     * @return void
+     */
+    public function handleAjaxMediaManager(Doku_Event $event)
+    {
+        if ($event->data !== 'plugin_move_rename_mediamanager') return;
+
+        if (!checkSecurityToken()) return;
```

Comment:
It might be nicer to display an actual error to the user to give the user at least a hint what the problem is, otherwise a problem with a wrong token is really hard to detect.